ravenframework/Models/PostProcessors/Validations/PPDSS.py

```diff
@@ -114,6 +136,18 @@ def _handleInput(self, paramInput):
         self.pivotParameterFeature = child.value
       elif child.getName() == 'pivotParameterTarget':
         self.pivotParameterTarget = child.value
+      elif child.getName() == 'separateFeatureData':
+        if 'type' not in child.parameterValues.keys():
+          self.raiseAnError(IOError, 'Tag separateFeatureData must have attribute "type"')
```

Comment:
In the manual, you mentioned, there is a default value for "type", please check it and make it consistent.

Reply:
Have removed the error message and now use the default.